### DISCUSSION TITLE
ARIA2 Is miner Malware?

### DISCUSSION THREAD

**Discussion by jinwolf2** [BlafKing/sd-civitai-browser-plus#303] [Q&A]
recently I have been having problems with a mining malware that always runs every time, the weird thing is that I only have this extension and aria2 being compiled I think it may be a reverse shell to take control of the machines or send scripts because it always installs dependencies that we never want. 
![Captura de pantalla 2024-06-19 a las 9 07 53](https://github.com/BlafKing/sd-civitai-browser-plus/assets/54153186/05744b81-2170-4686-a146-0be92bc3ab79)

**Comment by BenjaminSymons** [comment]
Can you explain how you got the above log output? Doesn't seem likely to be aria2
